config_flask.py

```diff
@@ -14,9 +15,10 @@ class BaseConfig(object):
     # Database
     POSTGRES_HOST = os.environ.get('POSTGRES_HOST', 'localhost')
     POSTGRES_DBNAME = os.environ.get('POSTGRES_DBNAME', 'gis')
-    POSTGRES_PORT = int(os.environ.get('POSTGRES_PORT', '5432'))
-    POSTGRES_USER = os.environ.get('POSTGRES_USER', 'gis_admin')
-    POSTGRES_PASS = os.environ.get('POSTGRES_PASS', 'admin')
+    POSTGRES_PORT = int(os.environ.get('POSTGRES_PORT', '5433'))
+    POSTGRES_USER = os.environ.get('POSTGRES_USER', 'gisops')
+    POSTGRES_PASS = os.environ.get('POSTGRES_PASS', 'gisops')
```

Comment:
Hm, just seeing this, since Travis failed. Can we revert this again? If you want local overrides, create a `.env` or put it in `.flaskenv`.

Reply:
Yep, done